query/src/main/scala/filodb/query/LogicalPlan.scala

```diff
@@ -70,6 +70,9 @@ sealed trait PeriodicSeriesPlan extends LogicalPlan {
   def endMs: Long
 
   def replacePeriodicSeriesFilters(filters: Seq[ColumnFilter]): PeriodicSeriesPlan
+
+  // The array of all at modifier
+  def atModifierTimestampsWithOffset: Seq[Long] = Seq()
```

Comment:
Double-checking: https://github.com/filodb/FiloDB/pull/1678#discussion_r1386119976

Can this default implementation be removed?

Reply:
<img width="656" alt="image" src="https://github.com/filodb/FiloDB/assets/120689245/3ee7f80c-8b9b-4bc6-a3ee-4397f2ca0ca2">
A few classes have to implement this functions to remove the default one